### PR TITLE
Ztri 15 workshop overview

### PR DESCRIPTION
**Pull request by jamesj-0**

### CODE REVIEW

**Comment by jamesj-0** [comment]
Completed the workshop overview pages

File: .functions/feedback-workshop/feedback-workshop.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const apiKey = process.env.AIRTABLE_KEY;
+const apiBaseId = process.env.AIRTABLE_BASE;
+const Airtable = require('airtable');
+
+var base = new Airtable({ apiKey }).base(apiBaseId);
+
+exports.handler = function (event, context, callback) {
+	const ID = event.queryStringParameters.id;
+	console.log('exports.handler -> ID', ID);
```
Comment (Joepock123) [inline .functions/feedback-workshop/feedback-workshop.js:9]: Smelly console.log. Use `npm run lint` to find these stinkers

File: src/components/Pages/workshopOverview/Comments.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import api from '../../../api/api';
+
+const CommentsWrapper = styled.section`
```
Comment (Joepock123) [inline src/components/Pages/workshopOverview/Comments.js:6]: You can separate out these styled components into another file

File: src/components/Pages/workshopOverview/MainSection.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+import PrimaryButton from '../../buttons/PrimaryButton';
+import SecondaryButton from '../../buttons/SecondaryButton';
+
+const MainWrapper = styled.section`
+	display: flex;
+	flex-direction: column;
+`;
+
+const ContentDiv = styled.div``;
```
Comment (Joepock123) [inline src/components/Pages/workshopOverview/MainSection.js:13]: Is this styled?

File: src/components/Pages/workshopOverview/TitleSection.js [hunk 4]
```diff
@@ -1,52 +1,81 @@
...
+// import vars from '../../../styles';
+import api from '../../../api/api';
 
 const Section = styled.section``;
 
-const TitleContainer = styled.div`
-	background: ${vars.sand};
+const Author = styled.p`
+	font-family: Roboto;
+	font-style: normal;
```
Comment (Joepock123) [inline src/components/Pages/workshopOverview/TitleSection.js:11]: Nice use of the variables.js file

File: src/components/Pages/workshopOverview/Comments.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+	display: flex;
+	flex-direction: column;
+`;
+
+const CommentsTitle = styled.h2`
+	font-family: Roboto;
+	font-style: normal;
+	font-weight: bold;
+	font-size: 23.43px;
```
Comment (Ivo-Evans) [inline src/components/Pages/workshopOverview/Comments.js:15]: :thinking: 
23.43px ??

**Pull request merged**